### ISSUE TITLE
some question with 'unique ID in the current frame'

### ISSUE THREAD

**Issue by Eyoahz1243fei** [OpenDriveLab/OpenLane-V2#107]
When submitting the test document, how should I fill in the information about "unique ID in the current frame"? I don't see any rules in the training set or the validation set.

**Comment by sephyli** [comment]
The "unique ID in the current frame" is an identifier with (split, segment_id, timestamp) format. This format is also used in the training and validation pickle files. For more details, please refer to [submission format](https://github.com/OpenDriveLab/OpenLane-V2/blob/master/docs/submission.md#driving-scene-topology).
